examples/plugins/afk.js

```diff
@@ -0,0 +1,42 @@
+// All plugins are loaded with the bot.loadPlugin function, look at documentation for more info
+// https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#botloadpluginplugin
+// Here is an example AFK plugin
+
+// Each plugin should have a inject function
+// This is the function that will be put in bot.loadPlugin
+function inject (bot, option) {
+  // create a scope for you functions
+  bot.afk = {}
+
+  let afkInterval, rotation
+  // This scope can be used to store some variables for user reference.
+  bot.afk.status = 'Idle'
+
+  // All your plugin functions should be in the scope other than the logic.
+  bot.afk.start = async () => {
+    afkInterval = setInterval(async () => {
+      if (rotation === 0) {
```

Comment:
this will never be true since you treat rotation as a boolean so using === will never work against a number